include/netlink/xfrm/template.h

```diff
@@ -65,7 +65,7 @@ extern int                      xfrmnl_user_tmpl_get_proto (struct xfrmnl_user_t
 extern int                      xfrmnl_user_tmpl_set_proto (struct xfrmnl_user_tmpl*, unsigned int);
 
 extern int                      xfrmnl_user_tmpl_get_family (struct xfrmnl_user_tmpl*);
-extern int                      xfrmnl_user_tmpl_set_family (struct xfrmnl_user_tmpl*, unsigned int);
+extern int                      xfrmnl_user_tmpl_set_family (struct xfrmnl_user_tmpl*, int);
```

Comment:
Sure, will do. I wasn't sure whether this is ABI-breaking or not, so I went for the solution I considered safer.